book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author, :rental
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
+  end
+
+  def add_rental(book)
+    @rental.push(book)
+    rental.book = self
```

Comment:
Please create a new rental in `#add_rental` of the `Book` class and pass the book as a `self`, person, and date as parameters. You may see what parameters are needed by Rental's class constructor to create an object in order to pass the right params.

Reply:
Hi, Thank you for understanding and making changes!